jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -115,98 +120,66 @@ export class MountPlugin implements IMountPlugin {
+    this._config.title.icon = settingsIcon;
+    this._config.title.className = 'pachyderm-config-tab';
+
+    this._pfsBrowser = createCustomFileBrowser(
+      app,
+      manager,
+      factory,
+      'pfs',
+      'explore',
+      'pfs',
+    );
 
-    this._mountedList = ReactWidget.create(
+    this._explore = ReactWidget.create(
       <UseSignal signal={this._poller.mountedSignal}>
         {(_, mounted) => (
-          <div className="pachyderm-mount-base">
-            <div className="pachyderm-mount-config-container">
-              <div className="pachyderm-mount-base-title pachyderm-mount-base-title-button-font">
-                Explore
-              </div>
-              <button
-                className="pachyderm-button-link"
-                data-testid="Datum__mode"
-                onClick={() => this.setShowDatum(true)}
-                style={{
-                  marginRight: '0.25rem',
-                }}
-              >
-                Test{' '}
-              </button>
-              <button
-                className="pachyderm-button-link"
-                onClick={() => this.setShowPipeline(true)}
-              >
-                Publish{' '}
-                <sup className="pachyderm-button-alpha-notice">Alpha</sup>
-              </button>
-              <button
-                className="pachyderm-button-link"
-                data-testid="Config__mode"
-                onClick={() => this.setShowConfig(true)}
-              >
-                <settingsIcon.react
-                  tag="span"
-                  className="pachyderm-mount-icon-padding"
-                />
-              </button>
-            </div>
-            <SortableList
-              open={this.openPFS}
-              items={mounted ? mounted : this._poller.mounted}
-              updateData={this._poller.updateData}
-              mountedItems={[]}
-              type={'mounted'}
-              projects={[]}
-            />
-          </div>
-        )}
-      </UseSignal>,
-    );
-    this._mountedList.addClass('pachyderm-mount-react-wrapper');
-
-    this._unmountedList = ReactWidget.create(
-      <UseSignal signal={this._poller.unmountedSignal}>
-        {(_, unmounted) => (
-          <UseSignal signal={this._poller.projectSignal}>
-            {(_, projects) => (
-              <div className="pachyderm-mount-base">
-                <div className="pachyderm-mount-base-title">
-                  Unmounted Repositories
-                </div>
-                <SortableList
-                  open={this.openPFS}
-                  items={unmounted ? unmounted : this._poller.unmounted}
-                  updateData={this._poller.updateData}
-                  mountedItems={this._poller.mounted}
-                  type={'unmounted'}
-                  projects={projects ? projects : this._poller.projects}
-                />
-              </div>
+          <UseSignal signal={this._poller.unmountedSignal}>
```

Comment:
I am still wrapping my head around these Signals, but is there a reason we have to have three separate signals for mounted, unmounted, and projects? Seems like if we just had some `ExploreState: { projects, mounted, unmounted }` we could remove on some of this code since they all trigger a re-render of Explore.

No strong preference here tbh just wondering if there is some benefit I am missing since I am still learning about Signals.

Reply:
I'm not sure, I don't think I was involved with the creation of these particular signals. I think coming back to this when we deal with mounted-ness and state in the explore view is best. I'll add it to the associated ticket.

Reply:
Sounds good to me 👍🏻